nexus/api.go

```diff
@@ -18,12 +18,11 @@ import (
 )
 
 // Package version.
-const version = "v0.0.11"
+const version = "v0.0.12"
```

Comment:
What is this version number? I don't see it in https://github.com/nexus-rpc/api. Do we need to start tagging version?

Reply:
I see. I wonder if we want a separate PR for that just before release, but if we know we're just about to release this small change as another version, ok.